meta-mender-core/classes/mender-bootimg.bbclass

```diff
@@ -30,7 +30,7 @@ IMAGE_CMD_bootimg() {
             mkfs.${MENDER_BOOT_PART_FSTYPE_TO_GEN} \
                 $force_flag \
                 "${WORKDIR}/boot.${MENDER_BOOT_PART_FSTYPE_TO_GEN}" \
-                ${MENDER_BOOT_PART_FSOPTS} $label_flag "BOOT"
+                ${MENDER_BOOT_PART_FSOPTS} $label_flag "${MENDER_BOOT_PART_LABEL}"
```

Comment:
@lluiscampos Why do you think to use the $MENDER_BOOT_PART_LABEL in the bootimg class too?

Reply:
blkid is case sensitive Changed to MENDER_BOOT_PART_LABEL ??= "${@bb.utils.contains('MENDER_BOOT_PART_FSTYPE', 'vfat', 'BOOT', 'boot', d)}"